packages/storybook-react/stories/react-editors/markdown-editor.tsx

```diff
@@ -191,12 +191,11 @@ export const DualEditor: React.FC = () => {
 };
 
 const extensions = () => [
-  new LinkExtension({ autoLink: true }),
```

Comment:
Due to extension priority, the link extension should generally be added first.

Otherwise you can end up with

`<a href="URL">some </a><strong><a href="URL">bold</a></strong><a href="URL"> text</a>`

Rather than 
`<a href="URL">some <strong>bold</strong> text</a>`

Reply:
Thank you I reordered the extensions